lib/ansible/module_utils/basic.py

```diff
@@ -422,7 +422,7 @@ def remove_values(value, no_log_strings):
             value = unicode(bytes_value, 'utf-8', errors='replace')
         else:
             value = bytes_value
-    elif isinstance(value, Sequence):
+    elif isinstance(value, (Sequence, type({}.keys()))):
```

Comment:
In py3.3 this is accessible from `collections.abc.KeysView`.  I would move this into the `try/except` logic farther up, where `Sequence` is built for older python.  Maybe create `SEQUENCE_TYPES` or such.